nix/kore-ghc8107.nix.d/kore.nix

```diff
@@ -1,674 +1,37 @@
-{ system, compiler, flags, pkgs, hsPkgs, pkgconfPkgs, errorHandler, config, ...
-}:
-{
-  flags = { threaded = true; };
-  package = {
-    specVersion = "2.2";
```

Comment:
Honestly no idea what happened here ... This is the result of running `nix/rematerialize.sh` with my branch from latest master

Reply:
I removed this commit (anyway the "rematerialize` ran in CI,  although I had tried to commit a consistent state  before)